tests/cli/test_rasa_export.py

```diff
@@ -0,0 +1,412 @@
+    [(3, 2), ("not-a-float", 5.5), (None, "no-float")],
+)
+def test_timestamp_error_exit(
+    minimum_timestamp: Optional[float], maximum_timestamp: Optional[float],
+):
+    args = argparse.Namespace()
+    args.minimum_timestamp = (
+        str(minimum_timestamp) if minimum_timestamp is not None else None
+    )
+    args.maximum_timestamp = (
+        str(maximum_timestamp) if maximum_timestamp is not None else None
+    )
+
+    with pytest.raises(SystemExit):
+        # noinspection PyProtectedMember
+        export._inspect_timestamp_options(args)
+
+
+def _write_endpoint_config_to_yaml(path: Path, data: Dict[Text, Any]) -> Path:
+    endpoints_path = path / "endpoints.yml"
+
+    # write endpoints config to file
+    io_utils.write_yaml_file(
+        data, endpoints_path,
+    )
+    return endpoints_path
+
+
+def test_get_available_endpoints(tmp_path: Path):
+    # write valid config to file
+    endpoints_path = _write_endpoint_config_to_yaml(
+        tmp_path, {"event_broker": {"type": "pika"}, "tracker_store": {"type": "sql"}}
+    )
+
+    # noinspection PyProtectedMember
+    available_endpoints = export._get_available_endpoints(str(endpoints_path))
+
+    # assert event broker and tracker store are valid, others are not
+    assert available_endpoints.tracker_store and available_endpoints.event_broker
+    assert not available_endpoints.lock_store and not available_endpoints.nlg
+
+
+# noinspection PyProtectedMember
+def test_get_event_broker_and_tracker_store_from_endpoint_config(tmp_path: Path):
+    # write valid config to file
+    endpoints_path = _write_endpoint_config_to_yaml(
+        tmp_path, {"event_broker": {"type": "sql"}, "tracker_store": {"type": "sql"}},
+    )
+
+    available_endpoints = export._get_available_endpoints(str(endpoints_path))
+
+    # fetching the event broker is successful
+    assert export._get_event_broker(available_endpoints)
+    assert export._get_rasa_tracker_store(available_endpoints)
+
+
+# noinspection PyProtectedMember
+def test_get_event_broker_from_endpoint_config_error_exit(tmp_path: Path):
+    # write config without event broker to file
+    endpoints_path = _write_endpoint_config_to_yaml(
+        tmp_path, {"tracker_store": {"type": "sql"}}
+    )
+
+    available_endpoints = export._get_available_endpoints(str(endpoints_path))
+
+    with pytest.raises(SystemExit):
+        assert export._get_event_broker(available_endpoints)
+
+
+# noinspection PyProtectedMember
+def test_get_tracker_store_from_endpoint_config_error_exit(tmp_path: Path):
+    # write config without event broker to file
+    endpoints_path = _write_endpoint_config_to_yaml(tmp_path, {})
+
+    available_endpoints = export._get_available_endpoints(str(endpoints_path))
+
+    with pytest.raises(SystemExit):
+        assert export._get_rasa_tracker_store(available_endpoints)
+
+
+@pytest.mark.parametrize(
+    "requested_ids,expected",
+    [("id1", ["id1"]), ("id1,id2", ["id1", "id2"]), (None, None)],
```

Comment:
another interesting case, no?
```suggestion
    [("id1", ["id1"]), ("id1,id2", ["id1", "id2"]), (None, None), ("", None)],
```